library/include/ck/library/reference_tensor_operation/cpu/reference_layernorm.hpp

```diff
@@ -90,10 +90,13 @@ struct ReferenceLayernorm : public device::BaseOperator
 
             for(int m = 0; m < M; ++m)
             {
+                AccDataType divisor =
+                    static_cast<AccDataType>(1 / ck::math::sqrt(var(m) + arg.epsilon_));
```

Comment:
`static_cast<AccDataType>(1)` rather than putting the whole expression inside `static_cast<AccDataType>(...)`.   I think `ck::math::sqrt(var(m) + arg.epsilon_)` is already a `ComputeDataType` expression

Reply:
Fix in 5490de184a3326d981e27a5e2f4b96358dc6979d